tests/unit-tests/preprocess.test.ts

```diff
@@ -1,8 +1,7 @@
 import { describe, expect, test } from 'vitest';
 
-import { codeToGlimmerAst } from '../../src/parse/index.js';
```

Comment:
Why include these changes in this pr?

I agree with them, but it's added noise in what the pr description lead me to believe was 'just dep updates'